crt_portal/cts_forms/migrations/0085_remove-assignee_20200716.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 2.2.13 on 2020-07-16 15:26
+
+from django.db import migrations
+from datetime import datetime
+import pytz
+from actstream import action
+from django.contrib.auth import get_user_model
+from actstream import registry
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cts_forms', '0084_form_template_standardization'),
+    ]
+
+    def remove_assignee(apps, schema_editor):
+        # get all reports
+        reports = apps.get_model('cts_forms', 'Report')
+        # user = apps.get_model('cts_forms', 'User')
```

Comment:
I like the usage of `get_user_model` here -- can we remove this comment?

Reply:
Yes, i'll remove commented out code.